snap/README.md

```diff
@@ -0,0 +1,73 @@
+# EdgeX Foundry App Service Configurable Snap
+[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgex-app-service-configurable)
+
+This folder contains snap packaging for the EdgeX Foundry's App Service Configurable application service.
+
+The project maintains a rolling release of the snap on the `edge` channel that is rebuilt and published at least once daily through the jenkins jobs setup for the EdgeX project. You can see the jobs run [here](https://jenkins.edgexfoundry.org/view/Snap/) specifically looking at the `edgex-app-service-configurable-snap-{branch}-stage-snap`.
+
+The snap currently supports both `amd64` and `arm64` platforms.
+
+## Installation
+
+### Installing snapd
+The snap can be installed on any system that supports snaps. You can see how to install 
+snaps on your system [here](https://docs.snapcraft.io/t/installing-snapd/6735).
```

Comment:
minor nitpick, but the preferred URLs for docs is now snapcraft.io/docs, so this should be i.e. https://snapcraft.io/docs/installing-snapd (but of course the old links _should_ keep working, but well you know how this sort of thing goes :wink: )

Reply:
OK, I'll update this. I just copied from the egdex-go README, so it probably needs updating as well...

Reply:
fair, it probably does